apps/src/templates/lessonOverview/ResourceList.jsx

```diff
@@ -85,7 +93,7 @@ export default class ResourceList extends Component {
       >
         {resource.name}
       </a>
-      {resource.type && ` -  ${resource.type}`}
+      {resource.type && ` -  ${resource.type} `}
```

Comment:
this extra space just adds some padding between the text and the new button. there didn't seem to be a way to do it in CSS that worked for both "button on the same line as text" and "button wrapped to new line" scenarios.